lib/std/json.zig

```diff
@@ -2648,17 +2673,9 @@ pub fn stringify(
             }
```

Comment:
Why is l2672 different from l2666?

Reply:
I don't know, but this is completely unrelated to this PR.